theme_test_custo/static/tests/tours/website_theme_preview.js

```diff
@@ -0,0 +1,41 @@
+/** @odoo-module */
+
+import tour from "web_tour.tour";
+
+tour.register("website_theme_preview", {
+    test: true,
+    url: "/web#action=website.action_website_configuration",
+},
+[{
+    content: "Click on create new website",
+    trigger: 'button[name="action_website_create_new"]',
+}, {
+    content: "insert website name",
+    trigger: '[name="name"] input',
+    run: "text Website Test",
+}, {
+    content: "Validate the website creation modal",
+    trigger: "button.btn-primary",
+},
+// Configurator first screen
+{
+    content: "Click Skip and start from scratch",
+    trigger: "button:contains('Skip and start from scratch')",
```

Comment:
This is usually not great to use wording to find elements in tests, we might change the wording or fix a typo later which would break the test.
It's fine tho, but wanted let you know that you could have avoided that by writing the test another way: simply enter edit mode, third tab, click on "Switch theme", which is another (and simpler/shorter) way to access this theme selection screen.